pyproject.toml

```diff
@@ -74,7 +74,15 @@ extend-exclude = '''
 '''
 
 [tool.pytest.ini_options]
-testpaths = "tests"
+minversion = "6.0"
+addopts = ["-ra", "--showlocals", "--strict-markers", "--strict-config"]
+xfail_strict = true
+filterwarnings = [
+    "error",
+    "ignore::Warning:_pytest.*",
```

Comment:
FYI, `_pytest._path` produces these encoding warnings. Probably could be solved by swapping out `tmpdir`/`tmpdir_factor` with `tmp_path` and `tmp_path_factory` - not sure there's much interest in maintaining (the private copy of) `py.path`.

Reply:
Does that mean this PR is obsolete now that I merged the pathlib migration PR?

Reply:
No, I was hoping this line could be removed after that PR, but there's one remaining `_pytest` file that produces an EncodingWarning:


```
/Users/henryschreiner/git/software/wheel/.tox/py310/lib/python3.10/site-packages/_pytest/faulthandler.py:30: EncodingWarning: 'encoding' argument not specified
  config.stash[fault_handler_stderr_key] = open(stderr_fd_copy, "w")
```


Strangely, with warnings as errors on, this error shows, but it doesn't seem to count as an error that will fail the tests. It appears before the "test session starts" line. Usually I thought this would be an INTERNAL ERROR, but here it's not. At least with the version of pytest I'm using.

So if you want, I _think_ this line could be removed. I think it's probably better just to hide errors from `_pytest` though, to avoid this stray message. I can also set it to "default", which will make it a normal non-failing warning.